### PR TITLE
feat: add user agent

### PR DESCRIPTION
**Pull request by BenjaminHalko**
Adds a user agent to all outgoing requests

closes: #1359

### CODE REVIEW

**Review by validcube — APPROVED** [review]
🥞 Approved

**Comment by PalmDevs** [comment]
Next PR: cookies stealer + trojan + tracker + sell data to darkweb :trollface:

**Comment by validcube** [comment]
> Next PR: cookies stealer + trojan + tracker + sell data to darkweb :trollface:

Note to team: Do not give PaLM the power to do anything related to public relation. :trollface:

**Review by oSumAtrIX — APPROVED** [review]

File: app/src/main/java/app/revanced/manager/di/HttpModule.kt [hunk 1]
```diff
@@ -1,9 +1,12 @@
 package app.revanced.manager.di
 
 import android.content.Context
+import app.revanced.manager.BuildConfig
 import io.ktor.client.*
 import io.ktor.client.engine.okhttp.*
+import io.ktor.client.plugins.BrowserUserAgent
```
Comment (Axelen123) [inline app/src/main/java/app/revanced/manager/di/HttpModule.kt:7]: This seems to be unused

**Pull request merged**